app/views/ViewDates.scala

```diff
@@ -16,13 +16,17 @@
 
 package views
 
+import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 import java.time.temporal.TemporalAccessor
 
 object ViewDates {
 
   val formatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm")
+  val dateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("d MMM uuu").withZone(ZoneId.systemDefault)
+  val timeFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("HH:mm").withZone(ZoneId.systemDefault)
 
   def format(temporal: TemporalAccessor): String = formatter.format(temporal)
+  def formatDateAtTime(temporal: TemporalAccessor): String = s"${dateFormatter.format(temporal)} at ${timeFormatter.format(temporal)}"
```

Comment:
This could be done with single formatter
```
DateTimeFormatter.ofPattern("d MMM uuu 'at' HH:mm").withZone(ZoneId.systemDefault)
```

Reply:
Didn't know you could do that (quote the literal text).  Thanks